packages/manager/apps/pci-block-storage/src/api/data/quota.ts

```diff
@@ -12,9 +14,40 @@ export interface RegionQuota {
 
 export const getRegionsQuota = async (
   projectId: string,
-): Promise<RegionQuota[]> => {
-  const { data } = await v6.get<RegionQuota[]>(
-    `/cloud/project/${projectId}/quota`,
+  region?: string,
+): Promise<RegionQuota> => {
+  const { data } = await v6.get<RegionQuota>(
+    `/cloud/project/${projectId}/region/${region}/quota`,
   );
   return data;
 };
+
+function isLocalZone(region: string) {
+  return region.split('-')[2] === 'LZ';
+}
+
+function getVolumeMaxSize(region: string) {
+  return isLocalZone(region) ? 4 * 1000 : 12 * 1000;
+}
```

Comment:
As a suggestion, we could move these functions into `utils` dedicated folder and add some test cases to ensure it works as expected.

Reply:
I prefer to have it in this file because in the best world it should be returned from the catalog or productAvailability (or the new route in development), it is only a local fix that shouldn't stay

Reply:
Understood 😌
Hope that will be reverted with the incoming API route :)